index.html

```diff
@@ -0,0 +1,426 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css" integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A==" crossorigin="anonymous" referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous" />
+    <title>Mine</title>
+    <style>
+        * {
+            padding: 0px;
+            margin: 0px;
+            box-sizing: border-box;
+        }
+        
+        .nav-container {
+            height: 4em;
+            display: flex;
+            align-items: center;
+            padding: 10px;
+            /* border: 1px solid green; */
+            box-shadow: 2px 1px 6px grey;
+        }
+        
+        .nav-list {
+            display: flex;
+            list-style: none;
+            align-items: center;
+            /* border: 1px solid salmon; */
+            width: 100%;
+        }
+        
+        .nav-item-3 {
+            flex: 1;
+        }
+        
+        form {
+            width: 100%;
+            display: flex;
+        }
+        
+        .search-input {
+            /* border: 1px solid fuchsia; */
+            flex: 1;
+        }
+        
+        .search-btn {
+            /* border: 1px solid sienna; */
+            width: 10%;
+        }
+        
+        .nav-menu {
+            display: none;
+        }
+        
+        .nav-item {
+            line-height: 1.2;
+            margin-right: 20px;
+            display: inline-block;
+            min-width: fit-content;
+            /* display: flex; */
+            /* height: 50px; */
+        }
+        
+        .nav-item-2 a {
+            text-decoration: none;
+            color: black;
+        }
+        
+        .btn-outline {
+            font-size: 0.85em;
+            font-weight: bold;
+            border: 1px solid black;
+            color: black !important;
+            padding: 0.7em 0.85em;
```

Comment:
you can add here`background: white` to match the button style in the requirements document